src/less/type-utilities.less

```diff
@@ -0,0 +1,81 @@
+
+@import (reference) "./fonts.less";
+
+/* Basscss Typography */
+
+.font-family-inherit { font-family: inherit }
+
+.font-size-inherit { font-size: inherit }
+
+.text--decoration-none { text-decoration: none }
```

Comment:
Considering they're written as classes, they can be used as mixins in less or as classes to be used inline.

If you're building a content heavy page, classes often make more sense than embedding in custom blocks of css for each kind of element or object element.

In your above example .my-link wouldn't need to exist as you would have .text--black and .text--decoration-none and you could then re-use text--black and text--decoration-none without having to re-write the css for every different element that may use those classes.